fbsimctl/FBSimulatorControlKit/Sources/Command.swift

```diff
@@ -280,6 +283,8 @@ extension Action {
       return (EventName.Diagnose, nil)
     case .WatchdogOverride(let bundleIDs, _):
       return (EventName.WatchdogOverride, ArraySubject(bundleIDs))
+    case .setLocation:
```

Comment:
This should be uppercase like the declaration of your enum. This is why you are [getting build errors on CI](https://travis-ci.org/facebook/FBSimulatorControl/jobs/149380233#L1823)